### PR TITLE
remove flash8 target (see #3922)

### PR DESCRIPTION
**Pull request by nadako**
Subj.

### CODE REVIEW

**Comment by nadako** [comment]
I'm not sure about the remoting changes, so it would be nice if someone could double-check it.

File: std/haxe/remoting/LocalConnection.hx [hunk 1]
```diff
@@ -107,11 +105,7 @@ class LocalConnection implements AsyncConnection implements Dynamic<AsyncConnect
 
 	#if flash
 	public static function connect( name : String, ?ctx : Context, ?allowDomains : Array<String> ) {
-		#if flash9
-			var l = new flash.net.LocalConnection();
-		#else
-			var l = new flash.LocalConnection();
-		#end
+		var l = new flash.LocalConnection();
```
Comment (Simn) [inline std/haxe/remoting/LocalConnection.hx]: this should be flash.net.LocalConnection

File: tests/unit/src/unit/TestRemoting.hx [hunk 2]
```diff
@@ -12,9 +12,9 @@ class TestRemoting extends Test {
 	static var _ : Dynamic = init();
 	static var ecnx : haxe.remoting.ExternalConnection;
 	static var ecnx2 : haxe.remoting.ExternalConnection;
-	static var ecnx3 : haxe.remoting.ExternalConnection;
+	// static var ecnx3 : haxe.remoting.ExternalConnection;
```
Comment (Simn) [inline tests/unit/src/unit/TestRemoting.hx]: Why are these commented out?

File: std/haxe/remoting/LocalConnection.hx (hunk 1 above)
Comment (nadako) [inline std/haxe/remoting/LocalConnection.hx]: Right, thanks!

File: tests/unit/src/unit/TestRemoting.hx (hunk 2 above)
Comment (nadako) [inline tests/unit/src/unit/TestRemoting.hx]: As far as I understood, they are somehow using flash8 iframe in unit.html or something and we removed that, but I'm not sure what to do with these tests.

Comment (Simn) [inline tests/unit/src/unit/TestRemoting.hx]: Oh well, I don't think the remoting tests have been executed this decade anyway.

Comment (nadako) [inline tests/unit/src/unit/TestRemoting.hx]: Yeah, though I'm kinda worried about that I could break something with this change, gotta review the diff again.

**Comment by nadako** [comment]
I double-checked the diff and it seems to be good. Are we merging this?

**Comment by Simn** [comment]
Since I'll have to shadow-release 3.2.0-rc1 again anyway we might as well do it. Just waiting for the CI to click the button.

**Comment by Simn** [comment]
I feel like this is historical moment!

**Comment by Justinfront** [comment]
seems to be missing from.
https://github.com/HaxeFoundation/haxe/wiki/Breaking-changes-in-Haxe-3.2.0

**Comment by Simn** [comment]
That's not a breaking change, it's a fix. :P

**Pull request merged**